files/aldap/prometheus.py

```diff
@@ -0,0 +1,46 @@
+import redis
+from datetime import datetime
+from aldap.parameters import Parameters
+from aldap.logs import Logs
+
+class Prometheus:
+
+    def __init__(self):
+        self.param = Parameters()
+        self.logs = Logs(self.__class__.__name__, False)
+
+        # Redis server
+        self.enabled = self.param.get('PROMETHEUS', False, bool)
```

Comment:
if any of these ENV variables is not set the aldap will start but it will throw an exception on every post request